.github/workflows/ci.yaml

```diff
@@ -26,14 +26,14 @@ jobs:
       - name: Install Python and set up Poetry
         uses: bakdata/ci-templates/actions/python-setup-poetry@v1.5.3
         with:
-          poetry-version: "1.5.1"
+          poetry-version: "1.4.0"
```

Comment:
is downgrading still necessary?

Reply:
yes

Reply:
could you link to the issue you created on the Poetry repo, please?

Reply:
don't ask me why but now it pass with 1.5.1 xD